code/modules/research/techweb/all_nodes.dm

```diff
@@ -803,10 +794,10 @@
 	id = "mech_laser_heavy"
 	display_name = "Exosuit Weapon (CH-LC \"Solaris\" Laser Cannon)"
 	description = "An advanced piece of mech weaponry"
-	prereq_ids = list("adv_beam_weapons")
+	prereq_ids = list("beam_weapons")
 	design_ids = list("mech_laser_heavy")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
-	export_price = 5000
+	export_price = 10000
```

Comment:
Unless /TG/ made export disks sell again, this is worthless as no nodes/tech disks sell.
Pr #35069